### PR TITLE
Documentation overhaul

### LINKED ISSUE #177: Document how to search products that are not referenced in eodag's catalog
eodag maintains a [providers configuration file](https://github.com/CS-SI/eodag/blob/develop/eodag/resources/providers.yml) that lists for each provider the products that eodag makes easily available to its users.

Each product referenced in the providers configuration file is also referenced in the [product types configuration file](https://github.com/CS-SI/eodag/blob/develop/eodag/resources/product_types.yml), which provides for each product type a set of metadata, such as :
```yaml
S3_SLSTR_L2LST:
  abstract: |
    The SLSTR Level-2 LST product provides land surface parameters generated on the wide 1 km measurement grid.
    It contains measurement file with Land Surface Temperature (LST) values with associated parameters (LST
    parameters are computed and provided for each pixel (re-gridded or orphan) included in the 1 km measurement grid)
  instrument: SLSTR
  platform: SENTINEL3
  platformSerialIdentifier: S3A,S3B
  processingLevel: L2
  sensorType: OPTICAL
  license: proprietary
  title: SENTINEL3 SLSTR Level-2 LST
  missionStartDate: "2016-02-16T00:00:00Z"
```

This metadata is added to the properties of the `EOProduct`s returned by a search. This config/catalog also allows to search for products that match one or more of these metadata (e.g. optical products with search(`sensorType="OPTICAL")`).

While eodag tries to support as many product types as possible, it happens sometimes that one would like to search for a product type not available in eodag's catalog. This use case is actually already supported!

This exemple shows how to search for Sentinel 1 GRD products, known as `S1_SAR_GRD` in eodag's catalog, with the provider `astraea_eod`:
```python
>>> from eodag import EODataAccessGateway
>>> dag = EODataAccessGateway()
#  product types supported by eodag for astraea_eod:
>>> [pt["ID"] for pt in dag.list_product_types(provider="astraea_eod")]
['GENERIC_PRODUCT_TYPE', 'L8_OLI_TIRS_C1L1', 'MODIS_MCD43A4', 'NAIP', 'S2_MSI_L1C', 'S2_MSI_L2A']
# providers supported by eodag that make S1_SAR_GRD products available
>>> dag.available_providers(product_type="S1_SAR_GRD")
['aws_eos', 'creodias', 'mundi', 'onda', 'peps', 'sobloo', 'wekeo']
```
The above snippet shows that this product isn't available with `astraea_eod`. However we can find on the [STAC Index](https://stacindex.org/catalogs/astraea-earth-ondemand#/5WpJuuvfexLDmorjkAGQ8X6oLkYte2k9UcLiQ6wm) that this product is actually available with `astraea_eod` and that its name is `sentinel1_l1c_grd`.

In that case we can use this collection name, unknown to eodag, to actually search for these products on `astraea_eod`:
```python
>>> dag.set_preferred_provider("astraea_eod")
>>> prods, _ = dag.search(productType="sentinel1_l1c_grd")
>>> prods[0]
EOProduct(id=S1A_IW_GRDH_1SDV_20190905T075654_20190905T075712_028883_034613_51EB, provider=astraea_eod)
```

* [x] This capability should be documented.
* [ ] Maybe, remove the `GENERIC_PRODUCT_TYPE` returned by `dag.list_product_types()`
* [x] Provide more details to developers/contributors of eodag about how to use/set `GENERIC_PRODUCT_TYPE`

### LINKED ISSUE #207: Rework the documentation, the tutorials in particular
We have more and more tutorials/examples and I have the feeling that this should motivate us to rework the documentation.

Right now we 6 tutorials that explain how to use eodag:
1. Eodag basics: configure PEPS, search with `.search`, serialize search results, display them on a map, download 1 product
2. Eodag advanced: more about configuration (e.g. with environment variable), select preferred provider, search with extra criteria (cloudCover), filter products with a cruncher, interactive map to select products to download, download all the products selected, explanation about ONLINE/OFFLINE
3. Eodag: define a search area. Dedicated to the different ways to define the search area with `geom` (WKT, Shapely, bounding box) and/or with a `locations` and how these two interact with each other. Lots of maps
4. EODAG as STAC client: dynamically add a provider, in this case a STAC provider; search for generic product types (L8 here); dynamically add a provider to search for products in a static STAC catalog, search products in such a catalog, clean them up to remove some assets not available (thumbnails) and download some products
5. NDVI computation with eodag and eodag-cube: simple configuration and search of products, post-process of these products with eodag-cube to compute the NDVI
6. Get Copernicus DEM using EODAG: download a grid file from AWS, dynamically add a custom provider for Copernicus DEM, create a GeoJSON from the grid that is compatible with eodag, plot the products on a map and filter them given a hand drawn polygon, download some products

And 3 tutorials that starts with using eodag to download some products and then give details about some advanced image processing workflows based on SNAP and Sen2Cor.

We have a *Python API* page that contains the auto-generated API documentation (from the module/class/function/method docstrings). There's also some additional text and explanation in there.  

I see some problems with this structure:
* It's certainly difficult for a user to find a specific piece of information in all the tutorials.
* There's quite some overlapping between these tutorials
* To properly learn everything about a single feature, it's likely than more than one tutorial has to be read
* It's difficult for us to maintain the tutorials. For instance I'd like to show how to use the new methods `search_iter_page` and `search_all`. It's not clear where I should add them.
* The pieces of information that are added in the *Python API* page (not the auto-documented one) is hard to find

After giving it some thoughts (I'm open for suggestions though!), I'd suggest that the main menu could be restructured to:
  * Introduction
  * Installation
  * Python user guide
    * Quickstart
    * Configure
    * Search
    * Crunch
    * Download
    * Post-process
    * Advanced usage
    * Examples
  * CLI user guide
  * STAC REST Interface
  * Python API
  * Contribute
  * Release history

Where we offer:
* *Quickstart*: a page that introduces the most important features of eodag in a simple workflow (configure, search, crunch, download) without getting into the details. It'd give a good overview about what eodag can do. I think it would be close to the current basics tutorial, although slightly extended.
* *Configuration*: how to configure eodag
* *Search*: how to search for products (geometry, locations, extra parameters, search/search_all/search_iter_page)
* *Crunch*: how to filter products with our "crunchers"
* *Download*: how to download products and what's going on when you do so (remote/local location, online/offline)
* *Post-process* : how to use a downloaded product with `eodag-cube`, or with another tool (e.g. SNAP)
* *Advanced Usage*: how to extend eodag thanks to its plugin architecture (maybe also some other advanced topics)
*  *Examples*: a page that links to multiple examples with some some specific usages. There we can add the ones that deal with SNAP and Sen2Cor. Although that I'd be in favor of dropping them at some point, these are more tutorials about how to use SNAP rather about how to use eodag, they were difficult to run and would certainly difficult to maintain if we need to. I would rather properly explain in the post-process page how downloaded products can be further processed. We could also add here a tutorial dedicated to how to map the results in a notebook, since we've been doing that a lot in the current ones.

Each of these pages (Quickstart, Configure, etc.) is a notebook of course. Everything that is added "manually" in the current *Python API* page would be explained in one of the User guide pages. It could then be removed entirely from there, the *Python API* page would just then include the auto-documented documentation.

### LINKED ISSUE #220: document needed pip version
Without upgrading pip on `ubuntu:18.04`, installation fails:
```bash
docker run --rm -it ubuntu:18.04 bash
apt update && apt install python3 python3-pip
python3 -m pip install eodag

[...]
Collecting pyproj (from eodag)
  Downloading https://files.pythonhosted.org/packages/2c/12/7a8cca32506747c05ffd5c6ba556cf8435754af0939906cbcc7fa5802ea3/pyproj-3.0.1.tar.gz (168kB)
    100% |################################| 174kB 1.7MB/s 
    Complete output from command python setup.py egg_info:
    ERROR: Cython.Build.cythonize not found. Cython is required to build pyproj.
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-cttccmsh/pyproj/
```
See in https://pyproj4.github.io/pyproj/stable/installation.html:
> Note: pyproj 3+ no longer supports manylinux1 wheels. pip>=19.0 is required to install manylinux2010 wheels.

Upgrading pip makes installation work again:
```bash
python3 -m pip install --upgrade pip
python3 -m pip install eodag

[...]
Successfully installed Jinja2-2.11.3 MarkupSafe-1.1.1 PyYAML-5.4.1 Werkzeug-1.0.1 attrs-20.3.0 boto3-1.17.43 botocore-1.20.43 certifi-2020.12.5 chardet-4.0.0 click-7.1.2 decorator-5.0.0 eodag-2.2.0 flasgger-0.9.5 flask-1.1.2 geojson-2.5.0 importlib-metadata-3.10.0 itsdangerous-1.1.0 jmespath-0.10.0 jsonpath-ng-1.5.2 jsonschema-3.2.0 lxml-4.6.3 markdown-3.3.4 mistune-0.8.4 owslib-0.23.0 ply-3.11 pyproj-3.0.1 pyrsistent-0.17.3 pyshp-2.1.3 pystac-0.5.6 python-dateutil-2.8.1 pytz-2021.1 requests-2.25.1 requests-futures-1.0.0 s3transfer-0.3.6 shapely-1.7.1 tqdm-4.59.0 typing-extensions-3.7.4.3 urllib3-1.26.4 usgs-0.3.0 whoosh-2.7.4 zipp-3.4.1
```

### PR DESCRIPTION
**Pull request by maximlt**
Fixes #207
Fixes #220 
Fixes #177 

The structure of the documentation has been reworked extensively:

* The landing page gives more information
* The left navigation bar is divided in two parts: User / Developer
* The former intro was turned into a larger Getting Started section with multiple pages (providers, product types, ...)
* Most of the tutorials were turned into an API User Guide section with multiple pages (configure, search, filter, ...). Each page describe how to use the main features of EODAG.
* The remaining tutorials were kept in a tutorials section
* A section is dedicated to STAC support with a Client page (the original tutorial) and a REST Server page (also the original one)
* The former Python reference has become an API reference (user-oriented) and a Plugins reference (developer) oriented

Since it felt like a new website, it required a new theme! I've found the [sphinx-book-theme](https://sphinx-book-theme.readthedocs.io/en/latest/index.html) very easy to set (good and concise documentation).

Regarding how the docs are built:

* I setup nbsphinx to run some notebooks (3 currently). I believe it's easier to maintain a documentation in sync with the code if the notebooks are run. However I didn't want to have nbsphinx run the notebooks that search for data or download data, that would just make building the docs too long, if that would even be possible.
* I pinned the docs dependencies (e.g. sphinx), I believe this can only help us. Let's update them from time to time.
* I added a configuration file for readthedocs (readthedocs.yml). I'd like to set readthedocs to fail whenever there is a warning, this should help to keep the docs well maintained.

Notebooks:
They are all in `docs/notebooks` and divided under `tutorials` and `api_user_guide`. There is a simple markdown page in this folder that links to the notebooks. Its aim is to be the landing page of the Binder badge in the README and on the index page. This structure would also make it quite easy for a user to save the notebooks and run them locally, or to give a live tutorial session. 

Some additional changes:
* CS website updated
* Cleaned up conf.py and .gitignore
* Update the contribution guideline
* Removed the burnt area detection tutorial with gpt (kept the one with snappy)
* Preferred folium over ipywidgets for plotting the products in the API user guide (it gave me some ideas for a .plot_map() method for SearchResult)

To complete this PR:

* [x] Fix the post-process notebook

Note for a next PR:

* ~~Rerun some of the notebooks to save their widget state (I'd like to wait until I have a better internet connection, possibly next Tuesday)~~. I wrote the notebooks with VS Code, it doesn't look as nice as with Jupyter Notebook or Lab but it's quite nice to have them open right next to the code. There is no way however to save the widget states. => I think this is really fine by now.

### CODE REVIEW

**Comment by maximlt** [comment]
I fixed the issue with the post-process notebook. I was passing an extent (i.e. crop window) to `get_data` that I believe was not fully contained within the product's extent.

I also addressed the comments that were made yesterday in our chat group.

**Comment by maximlt** [comment]
Recent updates:
* Add a note about installing pyproj with a too old version of pip (#220). A note here, if pyproj isn't necessary we should try to get rid of it, having it as a dependency can lead to lots of problems if one day eodag is used in a QGIS plugin
* Re-run the notebooks that required to save their state and widget

File: docs/getting_started_guide/configure.rst [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+
+Configure EODAG
+===============
+
+Configuration in ``eodag`` plays an important role:
+
+* To provide credentials (e.g. login/password, API key, etc.) required to download (and search sometimes) products with a provider
+* To specify the search priority of a provider
+* To indicate where to download products, whether to extract them or not, etc.
+* To dynamically update a provider or a new one
```
Comment (sbrunato) [inline docs/getting_started_guide/configure.rst:11]:
```suggestion
* To update a provider or add a new one
```

File: docs/getting_started_guide/configure.rst [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+   export EODAG__SOBLOO__DOWNLOAD__EXTRACT=True
+   export EODAG__SOBLOO__DOWNLOAD__OUTPUTS_PREFIX=/absolute/path/to/a/folder/
+
+
+Each configuration parameter can be set with an environment variable.
+
+CLI configuration
+^^^^^^^^^^^^^^^^^
+
+The commands that require a configuration file (e.g. ``download`` ) have an option
```
Comment (sbrunato) [inline docs/getting_started_guide/configure.rst:117]:
```suggestion
The command options that require a configuration file (e.g. ``eodag download`` ) have an argument
```

File: docs/getting_started_guide/configure.rst [hunk 3]
```diff
@@ -0,0 +1,231 @@
...
+   to a provider while trying to download a product. If the credentials are set, ``eodag`` will
+   keep going. An :class:`~eodag.utils.exceptions.AuthenticationError` is raised if the
+   authentication fails.
+
+   Make sure to check that your credentials are correctly set and to keep them up to date.
+
+Example
+^^^^^^^
+
+The following content can be saved to a file at ``/home/user/eodagworkspace/my_config.yml``.
```
Comment (sbrunato) [inline docs/getting_started_guide/configure.rst:182]:
```suggestion
Edit your configuration file ``$HOME/.config/eodag/eodag.yml`` with the following content:
```

File: docs/getting_started_guide/configure.rst [hunk 4]
```diff
@@ -0,0 +1,231 @@
...
+* The products should be downloaded to the folder ``/home/user/eodagworkspace/``.
+
+* An API key obtained from `sobloo` is saved there, it will be used to authenticate to the provider.
+
+This file can be used to download products with the API:
+
+.. code-block:: python
+
+   from eodag import EODataAccessGateway
+   dag = EODataAccessGateway("/home/user/eodagworkspace/my_config.yml")
```
Comment (sbrunato) [inline docs/getting_started_guide/configure.rst:211]:
```suggestion
   dag = EODataAccessGateway()
```

File: docs/getting_started_guide/configure.rst [hunk 5]
```diff
@@ -0,0 +1,231 @@
...
+   )
+   product_paths = dag.download_all(products)
+
+Or with the CLI:
+
+.. code-block:: console
+
+   cd /home/user/eodagworkspace/
+   eodag search -f my_config.yml \
+                -b 1 43 2 44 \
```
Comment (sbrunato) [inline docs/getting_started_guide/configure.rst:226]:
```suggestion
   eodag search -b 1 43 2 44 \
```

File: README.rst [hunk 6]
```diff
@@ -180,7 +185,7 @@ And browse http://127.0.0.1:5001:
    :width: 600px
 
 
-For more information, see `STAC REST interface usage <https://eodag.readthedocs.io/en/latest/use.html#stac-rest-interface>`_.
+For more information, see `STAC REST interface usage <https://eodag.readthedocs.io/en/stable/stac_rest.html>`_.
```
Comment (sbrunato) [inline README.rst:188]:
```suggestion
For more information, see `STAC REST API usage <https://eodag.readthedocs.io/en/stable/stac_rest.html>`_.
```

File: README.rst [hunk 7]
```diff
@@ -98,7 +103,7 @@ Example usage for interacting with the api in your Python code:
 
 This will search for Sentinel 2 level-1C products on the default provider and return the found products first page and
 an estimated total number of products matching the search criteria. And then it will download these products. Please
-check `tutorials <https://eodag.readthedocs.io/en/latest/tutos.html>`_ for more examples.
+check the `API User Guide <https://eodag.readthedocs.io/en/stable/api_user_guide.html>`_ for more details.
```
Comment (sbrunato) [inline README.rst:106]:
```suggestion
check the `Python API User Guide <https://eodag.readthedocs.io/en/stable/api_user_guide.html>`_ for more details.
```

File: docs/stac_rest.rst [hunk 8]
```diff
@@ -0,0 +1,107 @@
+.. _stac_rest:
+
+STAC REST Interface
+===================
```
Comment (sbrunato) [inline docs/stac_rest.rst:4]:
```suggestion
STAC REST API
=============
```

File: docs/stac_rest.rst [hunk 9]
```diff
@@ -0,0 +1,107 @@
...
+* ``year``
+        * ``month``
+                * ``day`` -> filters items using specified time interval
+* ``cloud_cover`` -> filters items with specified maximum cloud cover
+
+Example URLs:
+
+* http://127.0.0.1:5000/S2_MSI_L1C/country : lists available countries
+* http://127.0.0.1:5000/S2_MSI_L1C/country/FRA/year/2019/month/10/cloud_cover/10 : catalog referencing S2_MSI_L1C
+  products over France, aquired during October 2019, and having 10% maximum cloud cover
```
Comment (sbrunato) [inline docs/stac_rest.rst:91]: Add more examples (see README)

File: eodag/api/product/metadata_mapping.py [hunk 10]
```diff
@@ -117,16 +118,26 @@ def format_metadata(search_param, *args, **kwargs):
...
+        - ``to_iso_date``: remove the time part of a iso datetime string
         - ``remove_extension``: on a string that contains dots, only take the first
           part of the list obtained by splitting the string on dots
+        - ``get_group_name``: get the matching regex group name
+        - ``replace_str``: execute "string".replace(old, new)
+        - ``recursive_sub_str``: recursively substitue in the structure (e.g. dict)
+          values matching a regex
+        - ``slice_str``: slice a string (equivalent to s[start, end, step])
+        - ``fake_l2a_title_from_l1c``
+        - ``s2msil2a_title_to_aws_productinfo``
```
Comment (sbrunato) [inline eodag/api/product/metadata_mapping.py:140]:
```suggestion
        - ``fake_l2a_title_from_l1c``: used to generate SAFE format metadata for data from AWS
        - ``s2msil2a_title_to_aws_productinfo``: used to generate SAFE format metadata for data from AWS
```

**Review by sbrunato — CHANGES_REQUESTED** [review]

File: README.rst [hunk 11]
```diff
@@ -180,7 +185,7 @@ And browse http://127.0.0.1:5001:
    :width: 600px
 
 
-For more information, see `STAC REST interface usage <https://eodag.readthedocs.io/en/latest/use.html#stac-rest-interface>`_.
+For more information, see `STAC REST API usage <https://eodag.readthedocs.io/en/stable/stac_rest.html>`_.
 
 Command line interface
 ----------------------
```
Comment (sbrunato) [inline README.rst:191]: To be replaced with :

> Start playing with the CLI:
> - To search for some products:
```bash
eodag search --productType S2_MSI_L1C --box 1 43 2 44 --start 2021-03-01 --end 2021-03-31
```
> The request above searches for `S2_MSI_L1C` product types in a given bounding box, in March 2021. It saves the results in a GeoJSON file (`search_results.geojson` by default).

> Results are paginated, you may want to get all pages at once with `--all`, or search products having maximum 20% of coud cover with `--cloudCover 20`. For more information on available options:

```bash
eodag search --help
```
> - To download the result of the previous call to search:

```bash
eodag download --search-results search_results.geojson
```

> - To list all available product types and supported providers:
```
eodag list
```
>- To list available product types on a specified supported provider:
```
eodag list -p sobloo
```
>- To see all the available options and commands:
```
eodag --help
```
> - To print log messages, add `-v` to eodag master command. e.g. `eodag -v list`. The more `v` given (up to 3), the more verbose the tool is. For a full verbose output, do for example: `eodag -vvv list`

File: README.rst [hunk 12]
```diff
@@ -234,33 +239,45 @@ of this bbox"
...
-EODAG is developed by `CS GROUP - France <https://www.c-s.fr>`_.
+EODAG has been created by `CS GROUP - France <https://www.csgroup.eu/>`_.
+
+Contributors
+============
+
+New contributors to the project are listed below, we hope to see it growing fast!
 
+* @apparell
+* @ahuarte47
```
Comment (sbrunato) [inline README.rst:278]:
```suggestion
```

**Review by sbrunato — CHANGES_REQUESTED** [review]

File: README.rst (hunk 11 above)
Comment (maximlt) [reply README.rst:191]: Done in https://github.com/CS-SI/eodag/pull/233/commits/d507c08d46d13d199d80a757cc5e46544b718cfb

File: docs/notebooks/api_user_guide/1_overview.ipynb [hunk 13]
```diff
@@ -0,0 +1,4478 @@
...
+    "os.environ[\"EODAG__PEPS__AUTH__CREDENTIALS__USERNAME\"] = \"PLEASE_CHANGE_ME\"\n",
+    "os.environ[\"EODAG__PEPS__AUTH__CREDENTIALS__PASSWORD\"] = \"PLEASE_CHANGE_ME\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "A workspace directory is created to store the downloaded products."
+   ]
```
Comment (sbrunato) [inline docs/notebooks/api_user_guide/1_overview.ipynb:56]:
```suggestion
   "source": [
    "Now we will configure `eodag` to be able to download using *PEPS*. For that we need to fill our credentials:\n",
    "- in the user configuration file `~/.config/eodag.eodag.yml`:\n",
    "    ```yaml\n",
    "    peps:\n",
    "        priority: # Lower value means lower priority (Default: 1)\n",
    "        search:  # Search parameters configuration\n",
    "        download:\n",
    "            extract:  # whether to extract the downloaded products (true or false).\n",
    "            outputs_prefix: # where to store downloaded products.\n",
    "            dl_url_params:  # additional parameters to pass over to the download url as an url parameter\n",
    "        auth:\n",
    "            credentials:\n",
    "                username: PLEASE_CHANGE_ME\n",
    "                password: PLEASE_CHANGE_ME\n",
    "    ```\n",
    "- or using environment variables: (we also set `outputs_prefix`, the directory where to store downloaded products)"
   ]
```

File: docs/notebooks/api_user_guide/1_overview.ipynb [hunk 14]
```diff
@@ -0,0 +1,4478 @@
...
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "workspace = 'eodag_workspace_overview'\n",
+    "if not os.path.isdir(workspace):\n",
+    "    os.mkdir(workspace)"
+   ]
```
Comment (sbrunato) [inline docs/notebooks/api_user_guide/1_overview.ipynb:67]:
```suggestion
   "source": [
    "import os\n",
    "os.environ[\"EODAG__PEPS__AUTH__CREDENTIALS__USERNAME\"] = \"PLEASE_CHANGE_ME\"\n",
    "os.environ[\"EODAG__PEPS__AUTH__CREDENTIALS__PASSWORD\"] = \"PLEASE_CHANGE_ME\"\n",
    "os.environ[\"EODAG__PEPS__DOWNLOAD__OUTPUTS_PREFIX\"] = os.path.abspath(workspace)"
   ]
```

File: docs/notebooks/api_user_guide/1_overview.ipynb [hunk 15]
```diff
@@ -0,0 +1,4478 @@
...
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "os.environ[\"EODAG__PEPS__DOWNLOAD__OUTPUTS_PREFIX\"] = os.path.abspath(workspace)"
+   ]
+  },
```
Comment (sbrunato) [inline docs/notebooks/api_user_guide/1_overview.ipynb:84]:
```suggestion
```

File: docs/notebooks/api_user_guide/1_overview.ipynb [hunk 16]
```diff
@@ -0,0 +1,4478 @@
...
+    "            extract:  # whether to extract the downloaded products (true or false).\n",
+    "            outputs_prefix: # where to store downloaded products.\n",
+    "            dl_url_params:  # additional parameters to pass over to the download url as an url parameter\n",
+    "        auth:\n",
+    "            credentials:\n",
+    "                username: PLEASE_CHANGE_ME\n",
+    "                password: PLEASE_CHANGE_ME\n",
+    "    ```\n",
+    "- or using environment variables: (we also set `outputs_prefix`, the directory where to store downloaded products)"
+   ]
```
Comment (sbrunato) [inline docs/notebooks/api_user_guide/1_overview.ipynb:71]: messy display, please fix formatting

**Review by sbrunato — CHANGES_REQUESTED** [review]

**Review by sbrunato — APPROVED** [review]

**Pull request merged**